bin/alluxio

```diff
@@ -187,20 +186,16 @@ function main {
   COMMAND=$1
   shift
 
-  # bootstrap to create conf/alluxio-env.sh. This commands needs to be processed
-  # before sourcing libexec/alluxio-conf.sh which requires conf/alluxio-env.sh
-  if [[ "${COMMAND}" == "bootstrapConf" || "${COMMAND}" == "bootstrap-conf" ]]; then
-    bootstrapConf "$@"
-    exit 0
-  fi
-
   DEFAULT_LIBEXEC_DIR="${BIN}"/../libexec
   ALLUXIO_LIBEXEC_DIR=${ALLUXIO_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
   . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
 
   PARAMETER=""
 
   case ${COMMAND} in
+  "bootstrapConf")
```

Comment:
We are no longer supporting `bootstrap-conf` is that right?

Reply:
right. I don't remember why we want to support `bootstrap-conf` in addition to `bootstrapConf`. do you recall?